sormas-backend/src/main/java/de/symeda/sormas/backend/caze/classification/CaseClassificationFacadeEjb.java

```diff
@@ -509,6 +511,10 @@ private void buildCriteria() {
 			noneOf(anyOfSymptoms(SymptomState.NO, Disease.CORONAVIRUS)),
 			noneOf(suspect));
 
+		ClassificationCriteriaDto notACase = configFacade.isConfiguredCountry(CountryHelper.COUNTRY_CODE_LUXEMBOURG)
+			? allOf(negativeTestResult(Disease.CORONAVIRUS))
```

Comment:
allOf not needed because you have a single condition

Reply:
removed